asm/daxpy_amd64.s

```diff
@@ -136,3 +136,67 @@ V2:
 
 E2:
 	RET
+
+// func DaxpyIncTo(dst []float64, incDst, idst uintptr, alpha float64, x, y []float64, n, incX, incY, ix, iy uintptr)
+TEXT ·DaxpyIncTo(SB), NOSPLIT, $0
+	MOVQ   dst+0(FP), R10
+	MOVQ   incDst+24(FP), R13
+	MOVQ   idst+32(FP), BP
+	MOVHPD alpha+40(FP), X7
+	MOVLPD alpha+40(FP), X7
+	MOVQ   x+48(FP), R8
+	MOVQ   y+72(FP), R9
+	MOVQ   n+96(FP), DX
+	MOVQ   incX+104(FP), R11
+	MOVQ   incY+112(FP), R12
+	MOVQ   ix+120(FP), SI
+	MOVQ   iy+128(FP), DI
+
+	MOVQ SI, AX  // nextX = ix
+	MOVQ DI, BX  // nextY = iy
+	MOVQ BP, CX  // nextDst = idst
+	ADDQ R11, AX // nextX += incX
+	ADDQ R12, BX // nextY += incY
+	ADDQ R13, CX // nextDst += incDst
+	SHLQ $1, R11 // incX *= 2
+	SHLQ $1, R12 // incY *= 2
+	SHLQ $1, R13 // incDst *= 2
+
+	SUBQ $2, DX // n -= 2
+	JL   V3     // if n < 0 goto V2
+
+U3:  // n >= 0
```

Comment:
Yes, asmfmt gives this formatting but it is true that having one space less would be nicer (this particular comment is not very useful, I left it just to keep the changes at minimum). I will look into it.